frontend/src/features/help/components/HelpModal.tsx

```diff
@@ -108,6 +103,10 @@ const ModalStyled = styled(Modal)`
     text-shadow: 0 1px 0 #000;
     opacity: 0.9;
   }
+  .modal-body {
+    max-height: calc(100vh - 200px);
```

Comment:
this will detect small screens and prevent the help modal from stretching below the bottom of the screen.